### PR TITLE
Added wiring to set running user credentials

### PR DESCRIPTION
**Pull request by booms8**

### CODE REVIEW

File: src/server/tasks/3-install-machine.js [hunk 1]
```diff
@@ -50,11 +50,20 @@ function TaskImpl(options) {
...
+        log('setting the install update location to ' + path);
+        await client.setUpdatePath(path);
+      }
+
+      if(this.region.serviceConfig.runasUser && this.region.serviceConfig.runasPassword) {
+        let user = this.region.serviceConfig.runasUser;
+        let pass = this.region.serviceConfig.runasPassword;
+        log('setting the master user to ' + user);
+        await client.setMasterCredentials({'MasterRunnerUser': user, 'MasterRunnerPass': pass});
+      }
```
Comment (jakebrown58) [inline src/server/tasks/3-install-machine.js]: Please add a test to the 3-install-many test to cover this situation.

**Comment by booms8** [comment]
I'm unable to write tests because running tests for this project is failing on all our machines. I've rolled back to old versions of the code (before the Summa team even arrived) so I know it's not the present version. Any thoughts?

**Comment by jakebrown58** [comment]
talk to Adam Berg.

**Comment by jakebrown58** [comment]
You can drill into the travis-ci on the 'Details' link.  It's complaining about:
     TypeError: Cannot read property 'args' of null
      at test/server/tasks/3-install-machine.js:178:22

**Comment by booms8** [comment]
That's because I checked in an unfinished test so as not to lose my work. According to Adam it's a known issue with npm run test:server; npm run test:watch works fine so we're using that.

**Comment by jakebrown58** [comment]
ok - that makes sense.  can you update whatever you need to update in this branch to get the travis build green again?

File: src/server/models/region.js [hunk 2]
```diff
@@ -10,7 +10,9 @@ schema = {
     { "name": "regionName", "required": true },
     { "name": "regionDesc" },
     { "name": "serviceConfig" },
-    { "name": "browseConfig" }
+    { "name": "browseConfig" },
+    { "name": "runasUser"},
```
Comment (jakebrown58) [inline src/server/models/region.js]: I think we can exclude these changes to the region model.

**Pull request merged**